pkg/api/routes/inventory.go

```diff
@@ -10,9 +11,13 @@ import (
 )
 
 func processInband(c *fleetdb.Client, srvID uuid.UUID, dev *common.Device, log *zap.Logger) error {
+	log.Info("processing", zap.String("server id", srvID.String()), zap.String("device", dev.Serial))
+	fmt.Printf("not implemented for client %v", c)
```

Comment:
We shouldn't log anything to the console (i.e. `fmt.Printf`). The `Info` logs are fine for now but they should get turned down to `Debug` or `Trace` later on.